packages/wallets/src/features/cfd/components/ModalTradeWrapper/ModalTradeWrapper.tsx

```diff
@@ -7,8 +7,57 @@ import './ModalTradeWrapper.scss';
 import { WalletText } from '../../../../components/Base';
 import QRCode from 'qrcode.react';
 import useDevice from '../../../../hooks/useDevice';
+import { TMarketTypes, TPlatforms } from '../../../../types';
+import { PlatformToTitleMapper } from '../../constants';
 
-const ModalTradeWrapper: FC<PropsWithChildren<unknown>> = ({ children }) => {
+const LinksMapper: Record<
```

Comment:
Can move this to constant ? Having this in a Modal is weird 😸

Reply:
These constants are likely not going to be reused anywhere in the codebase, its just only for trade modal 😅

Reply:
The reason I didn't move these into the constants file is cuz I didn't want to pollute the constants file with rarely used constants, imo these should stay in the component file itself that's using it

Reply:
Then alrighty. If we want to use it multiple places in the future we can extract it out 👍